staging/vhost-device-gpu/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Red Hat Inc
+//
+// SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
+
+pub mod device;
+pub mod protocol;
+pub mod virtio_gpu;
+
+use std::path::PathBuf;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum GpuMode {
+    ModeVirglRenderer,
+    ModeGfxstream,
```

Comment:
```suggestion
    VirglRenderer,
    Gfxstream,
```
The mode is implied by the enum name already but that is just me nitpicking :)

Reply:
This is a clippy lint too; <https://rust-lang.github.io/rust-clippy/master/index.html#enum_variant_names>

Which reminds me that it'd be a good idea to add `deny` for some clippy lints like in other crates e.g. <https://github.com/rust-vmm/vhost-device/blob/main/vhost-device-sound/src/lib.rs>. It helps you save time for trivial stuff like this

Reply:
Thank you!